From: Anton Blanchard <anton@samba.org>
Subject: [PATCH] powerpc: add num_pmcs to 970MP cputable entry
References: 172957
Patch-mainline: yes

The 970MP cputable entry needs a num_pmcs entry for oprofile to work.

Signed-off-by: Anton Blanchard <anton@samba.org>
Signed-off-by: Olaf Hering <olh@suse.de>
---

---
 arch/powerpc/kernel/cputable.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-2.6.16/arch/powerpc/kernel/cputable.c
===================================================================
--- linux-2.6.16.orig/arch/powerpc/kernel/cputable.c
+++ linux-2.6.16/arch/powerpc/kernel/cputable.c
@@ -230,6 +230,7 @@ struct cpu_spec	cpu_specs[] = {
 			PPC_FEATURE_HAS_ALTIVEC_COMP,
 		.icache_bsize		= 128,
 		.dcache_bsize		= 128,
+		.num_pmcs		= 8,
 		.cpu_setup		= __setup_cpu_ppc970,
 		.oprofile_cpu_type	= "ppc64/970",
 		.oprofile_type		= PPC_OPROFILE_POWER4,
